src/containers/Main/Tabs/Screens/Home/index.tsx

```diff
@@ -21,13 +22,19 @@ const Home: FC<Props> = () => {
           <HeaderContainer>
             <Title>Learn</Title>
             <CategoryBar
-              data={['cat1', 'cat2', '3', '4', '5', '6']}
+              data={allCategoryList}
               showsHorizontalScrollIndicator={false}
-              renderItem={() => <CategoryCard />}
+              renderItem={(category) => (
+                <CategoryCard
+                  category={category.item}
+                  filter={filter}
+                  setFilter={setFilter}
+                />
+              )}
               horizontal={true}
             />
           </HeaderContainer>
-          <LessonList categoryFlag={categorySelector} />
+          <LessonList isFiltered={filter} lessons={filteredList} />
```

Comment:
Same here. as `isFiltered` is a string, this should be:

```suggestion
          <LessonList filter={filter} lessons={filteredList} />
```